test/e2e_node/cgroup_manager_test.go

```diff
@@ -71,4 +72,164 @@ var _ = framework.KubeDescribe("Kubelet Cgroup Manager [Skip]", func() {
+						pod := &api.Pod{
+							ObjectMeta: api.ObjectMeta{
+								Name:      podName,
+								Namespace: f.Namespace.Name,
+							},
+							Spec: api.PodSpec{
+								Containers: []api.Container{
+									{
+										Image:   "gcr.io/google_containers/busybox:1.24",
+										Name:    contName,
+										Command: []string{"sh", "-c", "if [ -d /tmp/memory/pod#" + podUID + " ] && [ -d /tmp/cpu/pod#" + podUID + " ]; then exit 0; else exit 1; fi"},
+										VolumeMounts: []api.VolumeMount{
+											{
+												Name:      "sysfscgroup",
+												MountPath: "/tmp",
+											},
+										},
+									},
+								},
+								Volumes: []api.Volume{
+									{
+										Name: "sysfscgroup",
+										VolumeSource: api.VolumeSource{
+											HostPath: &api.HostPathVolumeSource{Path: "/sys/fs/cgroup"},
+										},
+									},
+								},
+							},
+						}
+						podClient := f.PodClient()
+						podClient.Create(pod)
+						err := framework.WaitForPodSuccessInNamespace(f.Client, podName, contName, f.Namespace.Name)
+						Expect(err).NotTo(HaveOccurred())
+					})
+				}
+			})
+		})
+		Context("On scheduling a BestEffort Pod", func() {
+			It("Pod containers should have been created under the BestEffort cgroup", func() {
+				if framework.TestContext.CgroupsPerQOS {
+					var podUID string
+					By("Creating a BestEffort pod in Namespace", func() {
+						podName := "qos-pod" + string(uuid.NewUUID())
+						contName := "qos-container" + string(uuid.NewUUID())
+						pod := &api.Pod{
+							ObjectMeta: api.ObjectMeta{
+								Name:      podName,
+								Namespace: f.Namespace.Name,
+							},
+							Spec: api.PodSpec{
+								// Don't restart the Pod since it is expected to exit
+								RestartPolicy: api.RestartPolicyNever,
+								Containers: []api.Container{
+									{
+										Image:     framework.GetPauseImageName(f.Client),
+										Name:      contName,
+										Resources: getResourceRequirements(getResourceList("", ""), getResourceList("", "")),
+									},
+								},
+							},
+						}
+						podClient := f.PodClient()
+						apiPod := podClient.Create(pod)
+						podUID = string(apiPod.UID)
+					})
+					By("Checking if the pod cgroup was created", func() {
+						podName := "qos-pod" + string(uuid.NewUUID())
+						contName := "qos-container" + string(uuid.NewUUID())
+						pod := &api.Pod{
+							ObjectMeta: api.ObjectMeta{
+								Name:      podName,
+								Namespace: f.Namespace.Name,
+							},
+							Spec: api.PodSpec{
+								// Don't restart the Pod since it is expected to exit
+								RestartPolicy: api.RestartPolicyNever,
+								Containers: []api.Container{
+									{
+										Image:   "gcr.io/google_containers/busybox:1.24",
+										Name:    contName,
+										Command: []string{"sh", "-c", "if [ -d /tmp/memory/BestEffort/pod#" + podUID + " ] && [ -d /tmp/cpu/BestEffort/pod#" + podUID + " ]; then exit 0; else exit 1; fi"},
```

Comment:
One thing I was thinking of is to get rid of this whole cgroup_manager_test file for now. as these tests won't be run once the flag is disabled by  #30475 .
We have already tested the code by running the e2e tests, and it shouldn't be a problem till the flag is disabled. WDYT?

If you think otherwise I can add a TODO to make it work on systemd. 

> Even better would be to hit kubelet /configz and get the cgroup-root value. Something for a later PR
Sounds Cool. (y)